release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -208,6 +208,11 @@ For more information, see xref:../networking/hardware_networks/switching-bf2-nic
 [id="ocp-4-13-storage"]
 === Storage
 
+[id="ocp-4-13-storage-lvms-in-zpt"]
+==== Support for {lvms} in GitOps ZTP
+
+In {product-title} {product-version}, you can add and configure {lvms-first} through GitOps ZTP. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install[Intalling LVM Storage by using the web console] and xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu[LVM Storage].
```

Comment:
Will also need a link to the actual ZTP procedure: https://github.com/openshift/openshift-docs/pull/57467